packages/components/src/inputs/src/TextField.tsx

```diff
@@ -64,11 +64,9 @@ export interface TextFieldProps extends StyledComponentProps<RACTextFieldProps>
 
     /**
      * This should only be used with the `showCharacterCount` prop.
-     * If `true`, the TextArea prevents the text from ever going over the max length.
-     * If `false`, the TextArea will allow the text to go over the max length, but it will add an error look tot he character count.
-     * @default true
+     * If `true`, the TextArea will allow the text to go over the max length, but it will add an error look to the character count.
```

Comment:
I guess this was a copy paste error before, sorry. it should says text field here